presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -30,6 +30,12 @@ Map Functions
 
     See also :func:`map_agg` and :func:`multimap_agg` for creating a map as an aggregation.
 
+.. function:: map() -> map<unknown, unknown>
```

Comment:
Put this version before the one that takes arguments (functions within a section are sorted, so this one would sort first).